CDCgov/NEDSS-Modernization
# CNFT2-1467 Manage Tabs - edit tab in modal

Description
Here we add the tab editing functionality to the Manage Tabs modal.
Also took this opportunity to do some refactor and add alert banners when adding a new tab as well.
https://www.figma.com/file/Uz9YzHbHQAubHpQTcsibA2/NBS-PageBuilder-2.0?node-id=3526%3A109147&mode=dev

Tickets
https://cdc-nbs.atlassian.net/browse/CNFT2-1467?atlOrigin=eyJpIjoiZGY3NGZlNmU5ODQ5NGNmNzhjMTM2NDZmYWFjZjlmNDkiLCJwIjoiaiJ9

Steps to verify
From Page Builder Pages Library:
1. Select a Page by clicking the Page name
2. In the Tabs menu bar, select `Manage Tabs`
3. Click on the `Edit` (pencil icon) button to open the details in edit mode
4. Create changes to the name or visibility
5. Click save to bring up a confirmation alert, the tab edits should be shown in the Tabs menu bar.
